src/library/scala/math/package.scala

```diff
@@ -58,7 +58,21 @@ package object math {
    *          logarithms.
    */
   def exp(x: Double): Double = java.lang.Math.exp(x)
+
+  /** Returns the natural logarithm of a `double` value.
+   * 
+   * The natural logarithm is the logarithm base Euler's number `e`.
+   *
+   *  @param  x the number to take the natural logarithm of
+   *  @return the value `ln(x)` where ln is the natural lograrithm
```

Comment:
Maybe just "the value `logₑ(x)`"?

Still technically incorrect of course because floating point arithmetic, but going to fully correct route you'd end up with something along the lines of "the value y where y is the smallest double value for which `logₑ(x)` >= y or where y is the largest double value for which `logₑ(x)` <= y, and `logₑ(x)` <= `logₑ(z)` if x <= z and `logₑ(x)` >= `logₑ(z)` if x >= z, or NaN if x < 0" which is likely more confusing than useful.